src/app/components/Item/ItemTable.jsx

```diff
@@ -10,25 +10,28 @@ const ItemTable = ({ items, bibId, getRecord }) => {
   }
 
   return (
-    <table className="nypl-basic-table">
-      <caption className="hidden">Item details</caption>
-      <thead>
-        <tr>
-          <th>Location</th>
-          <th>Call No.</th>
-          <th>Status</th>
-          <th>Message</th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        {
-          items.map((item, i) =>
-            <ItemTableRow key={i} item={item} bibId={bibId} getRecord={getRecord} />
-          )
-        }
-      </tbody>
-    </table>
+    <dl>
```

Comment:
This table is being used in multiple locations where it may not be in a definition list, so it's better if this is still just a `<table>` and where it gets instantiated that it is wrapped in a `<dl><dd>` there.